openedx/adg/lms/registration_extension/tests/test_models.py

```diff
@@ -1,26 +1,49 @@
 """
 Tests for all the models in the registration_extension app
 """
-import datetime
+from datetime import date, datetime
 
 import pytest
 
+from openedx.adg.lms.registration_extension.constants import NO_BIRTH_DATE_ADDED
+
 from .factories import ExtendedUserProfileFactory
 
 
+@pytest.fixture(name='extended_profile')
+def get_extended_user_profile():
+    """
+    Fixture that returns an ExtendedUserProfile
+    """
+    return ExtendedUserProfileFactory()
+
+
 @pytest.mark.parametrize('is_saudi_national, has_birthdate, expected_output', [
     (True, True, True), (True, False, False), (False, True, False), (False, False, False)
 ])
 @pytest.mark.django_db
-def test_is_saudi_national_and_has_birthdate_property(is_saudi_national, has_birthdate, expected_output):
+def test_is_saudi_national_and_has_birthdate_property(
+    is_saudi_national, has_birthdate, expected_output, extended_profile
+):
     """
     Test that `is_saudi_national_and_has_birthdate` property of the ExtendedUserProfile model returns
     False unless the extended profile has `saudi_national` as true and birthdate field is added
     """
-    extended_profile = ExtendedUserProfileFactory()
     extended_profile.saudi_national = is_saudi_national
 
     if has_birthdate:
-        extended_profile.birth_date = datetime.date(1990, 1, 1)
+        extended_profile.birth_date = date(1990, 1, 1)
 
     assert extended_profile.is_saudi_national_and_has_birthdate == expected_output
+
+
+@pytest.mark.parametrize('birth_date, expected_age', [
+    (None, NO_BIRTH_DATE_ADDED), (datetime(1990, 1, 1), 31)
+], ids=['no-birthdate-added', 'birthdate-added'])
```

Comment:
Let's break the lines as follows to increase the readability
```
@pytest.mark.parametrize(
    'birth_date, expected_age', 
    [
        (None, NO_BIRTH_DATE_ADDED), 
        (datetime(1990, 1, 1), 31)
    ], 
    ids=['no-birthdate-added', 'birthdate-added']
)

Reply:
This gives pycodestyle error: `E128 continuation line under-indented for visual indent`